main/guides/orchestration/how-orch-works.md

```diff
@@ -15,6 +15,10 @@ To achieve these goals, several foundational technologies and protocols have
 been developed, with the [Inter-Blockchain Communication (IBC) protocol](https://ibcprotocol.org/) being
 one of the most prominent.
 
+<br/>
+<img src="./assets/orch_contract_arch.png" width="100%" />
```

Comment:
any particular reason to switch from .svg to .png? .svg is more accessible - the text is searchable and selectable

Reply:
oic

Reply:
I think free version of Canva does not let you download SVG.

Back to SVG again for that image by converting the source slide you linked. There is another image on the landing page now (in PNG for now). I would probably tinkering a bit more, and perhaps add one more image too.

Reply:
> the text is searchable and selectable

@dckc how is the text searchable and selectable in the SVG image? I cannot do that in your uploaded SVG. I could not find DEX in the source of your image as well.

Reply:
I thought I already fixed that?

https://8abcef3f.documentation-7tp.pages.dev/guides/orchestration/how-orch-works.html